libs/methodologies/bold/rule-processors/credits/src/rewards-distribution/rewards-distribution.helpers.ts

```diff
@@ -0,0 +1,182 @@
+import { sumBigNumbers } from '@carrot-fndn/shared/helpers';
+import { RewardsDistributionActorType } from '@carrot-fndn/shared/methodologies/bold/types';
+import {
+  type NonEmptyString,
+  type NonZeroPositive,
+} from '@carrot-fndn/shared/types';
+import BigNumber from 'bignumber.js';
+
+import type {
+  ActorsByActorType,
+  AggregateMassIdRewards,
+  Remainder,
+  ResultContentsWithMassIdCertificateValue,
+  RewardsDistribution,
+} from './rewards-distribution.types';
+
+export const formatPercentage = (percentage: BigNumber): BigNumber =>
+  percentage.dividedBy(100);
+
+export const formatDecimalPlaces = (value: BigNumber): BigNumber =>
+  value.decimalPlaces(6, BigNumber.ROUND_DOWN);
+
+export const calculateCreditPercentage = (
+  amount: BigNumber,
+  amountTotal: BigNumber,
+): string =>
+  (amountTotal.isZero()
+    ? new BigNumber(0)
+    : formatDecimalPlaces(amount.dividedBy(amountTotal).multipliedBy(100))
+  ).toString();
+
+export const calculateRemainder = (options: {
+  actors: ActorsByActorType;
+  creditsDocumentUnitPrice: NonZeroPositive;
+  massIdTotalValue: BigNumber;
```

Comment:
Here and in other helpers, I'm not feeling comfortable in having numeric values with different units. 
What do you think of normalizing the helpers to always receive BigNumber as input? So we move the responsibility using the correct number format to the upper scope.